packages/components/src/menu-item/index.js

```diff
@@ -21,7 +21,7 @@ import IconButton from '../icon-button';
  *
  * @return {WPElement} More menu item.
  */
-function MenuItem( { children, className, icon, onClick, shortcut, isSelected = false } ) {
+function MenuItem( { children, className, icon, onClick, shortcut, isSelected, role = 'menuitem', ...props } ) {
```

Comment:
Minor: I guess if we remove the role explicit prop and just use `role="menuitem"` in JSX, it has the exact same effect. Not sure if it's better or if we want to be explicit.

Reply:
My thinking was that `menuitem` would usually be the right role, and at least this way there's some out of the box accessibility. Perhaps it's worth adding something to the docs explaining the preferred usage, and also commenting in the code.